kinto/tests/test_config.py

```diff
@@ -68,7 +74,9 @@ def test_init_postgresql_values(self, mocked_render_template):
             'permission_backend': 'cliquet.permission.postgresql',
             'storage_url': postgresql_url,
             'cache_url':  postgresql_url,
-            'permission_url': postgresql_url
+            'permission_url': postgresql_url,
+            'kinto_version': __version__,
+            'config_file_timestamp': strftime('%a, %d %b %Y %H:%M:%S %z')
```

Comment:
nitpick: for the tests you can use a hard-coded version if you don't test its output